tests/playbooks/tests_route_device.yml

```diff
@@ -0,0 +1,202 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Test configuring the output device of the route
```

Comment:
```suggestion
- name: Test configuring the output device of routes
```

The article `the` is not correct here, since this is not about one specific route.